### PR TITLE
[Sprint 8] Fix time sheet ui

### PR DESCRIPTION
**Pull request by nguyenminhduong**

### CODE REVIEW

**Review by NguyenTienVinh — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -253,5 +253,11 @@
     <string name="total_fining">Total Fining</string>
     <string name="summary">Summary</string>
     <string name="handler">Handler :</string>
+    <string name="wo_il_le_lo_enough_compensation_time">WO/IL/LE/LO and enough compensation time</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:256]: FATAL: [MissingTranslation] "`wo_il_le_lo_enough_compensation_time`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -253,5 +253,11 @@
...
+    <string name="holiday">Holiday</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:257]: FATAL: [MissingTranslation] "`holiday`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -253,5 +253,11 @@
...
+    <string name="l_lo_le_forgot_card_forgot_checkin_checkout">L/LO/LE/forgot card/forgot checkin/checkout</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:258]: FATAL: [MissingTranslation] "`l_lo_le_forgot_card_forgot_checkin_checkout`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -253,5 +253,11 @@
...
+    <string name="no_data_not_enough_working_time">No Data/Not enough working time</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:259]: FATAL: [MissingTranslation] "`no_data_not_enough_working_time`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -253,5 +253,11 @@
...
+    <string name="p_paid_leave_of_absence">P: Paid leave of absence</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:260]: FATAL: [MissingTranslation] "`p_paid_leave_of_absence`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -253,5 +253,11 @@
...
+    <string name="ro_unpaid_leave_of_absence">Ro: Unpaid leave of absence</string>
```
Comment (daolq3012) [inline app/src/main/res/values/strings.xml:261]: FATAL: [MissingTranslation] "`ro_unpaid_leave_of_absence`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

**Pull request merged**